python/py-dmgbuild/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-dmgbuild
+version             1.6.2
+
+categories-append   devel
+license             {MIT PSF}
+platforms           {darwin any}
+supported_archs     noarch
+maintainers         nomaintainer
+description         macOS command line utility to build disk images
+long_description \
+    dmgbuild is a command line tool to create macOS disk images (aka .dmg files). While it is possible to create disk images easily enough from the command line using the hdiutil program that ships with macOS, there is no easy way to configure the appearance of the resulting disk image when the user opens it.
+
+homepage            https://dmgbuild.readthedocs.io/en/latest/
+
+checksums           md5     cc46f3a49ebbd014c83b6dd6faa16377 \
+                    rmd160  0a9436e0a4f98d95c031db32a5323f547533b4da \
+                    sha256  dba01d29f10c6804f2d72301600ddd03724daa41cd21a95409c44a3e199b19aa \
+                    size    36893
+
+patchfiles-append   patch-pyproject.patch
+
+python.versions     38 39 310 311 312
+python.pep517       yes
+
+if {$subport ne $name} {
+    depends_lib     port:py${python.version}-mac-alias \
```

Comment:
please use "append" as not to overwrite the pythonXY dependency added by the PortGroup